src/liquid/components/ld-input/readme.md

```diff
@@ -95,23 +91,15 @@ By default, the `ld-input` component is of [type `text`](https://developer.mozil
     id="focusable-disabled-input-1">
 </div>
 
-<div class="ld-input ld-input--disabled">
-  <input
-    placeholder="Placeholder"
-    value="Value"
-    aria-disabled="true"
-    id="focusable-disabled-input-2">
-</div>
-
 <!-- Example code for input prevention on aria-disabled input elements -->
 <script>
-  const inputs = document.querySelectorAll('#focusable-disabled-input-1, #focusable-disabled-input-2')
-  Array.from(inputs).forEach(input => {
-    const initialValue = input.value
-    input.addEventListener('input', () => {
-      input.value = initialValue
-    })
+;(() => {
+  const input = document.currentScript.previousElementSibling.querySelector('input')
+  const initialValue = input.value
+  input.addEventListener('input', () => {
```

Comment:
Definitely different situation but you mentioned to prefer the use of ld... event over the originals. Would it make sense here to also switch to the ldinput event?

Reply:
We do not generally prefer the use of ld*-events, only for non-composed events that do not bubble across the borders of the shadow DOM, we need to supply a prefixed surrogate event. (This is necessary because React "swallows" custom events that are named like native events - at least in case of non-composed events, afaik.)